app/javascript/helpers/data.js

```diff
@@ -120,3 +120,8 @@ export const customerDetailFields = [
     type: 'radio'
   }
 ];
+
+export const incomeLessThan5kField  = {
+  id: 'incomeLessThan5k',
+  label: 'How did you support yourself on less than $5000?'
```

Comment:
Is this really the label?

Reply:
![image](https://user-images.githubusercontent.com/22373872/60141307-fd662180-9808-11e9-94d6-7489d43c5323.png)

Reply:
Huh, ok then. Interesting.